packages/data-controls/src/index.ts

```diff
@@ -35,45 +37,67 @@ export function apiFetch( request ) {
  * Control for resolving a selector in a registered data store.
  * Alias for the `resolveSelect` built-in control in the `@wordpress/data` package.
  *
- * @param {Array} args Arguments passed without change to the `@wordpress/data` control.
+ * @param storeNameOrDescriptor
```

Comment:
Could be nice to have a description specified here. Something like "Name of the store, or store descriptor".